test/test_cuda.py

```diff
@@ -2585,7 +2585,12 @@ def test_rpow(self):
         _TestTorchMixin._test_rpow(self, lambda x: x.cuda())
 
     def test_int_pow(self):
-        _TestTorchMixin._test_int_pow(self, lambda x: x.cuda())
+        _TestTorchMixin._test_integral_pow(self, lambda x: x, torch.int8, (-3, 4))
+        _TestTorchMixin._test_integral_pow(self, lambda x: x, torch.uint8, (0, 4))
+        _TestTorchMixin._test_integral_pow(self, lambda x: x, torch.int16, (-5, 5))
+        _TestTorchMixin._test_integral_pow(self, lambda x: x, torch.int64, (-10, 10))
+        if not IS_WINDOWS:
```

Comment:
Can we have comment while it fails for WIN

Reply:
Addressed